### DISCUSSION TITLE
How to Remove Random id="irro" from code

### DISCUSSION THREAD

**Discussion by tlrcochran** [GrapesJS/grapesjs#5004] [Q&A]
Hello,

I have an issue where I am using custom blocks/components of code and when I go to view the code or export it, it adds random ID's to my div's/tables that do not have a class on them. It also adds box-sizing and border-box to everything.

I managed to fix the box-sizing and border-box, but is there anyway to prevent the random IDs from being added? 
![Screen Shot 2023-03-22 at 11 52 24 PM](https://user-images.githubusercontent.com/128666710/227099635-c40d8345-b709-4afd-8bdf-9dfd1b2515e1.png)

**Comment by artf** [comment]
You can use [toHTML with attributes option](https://grapesjs.com/docs/api/component.html#tohtml) and remove all the ids

**Reply by tlrcochran** [reply]
Thank you @artf. The documentation is helpful. However, when I use the code below to remove all of the ID's, it works in removing the id's added, but it also removes all inlined CSS from my html. Could you provide some insight into the best way to achieve removing all the ID's from the view code HTML without stripping the inline styling?

getInnerHTML({
 attributes(component, attributes) {
    if (component.get('tagName')) {
     attributes.id = false
   }
   return attributes;
 },
});
